astropy/units/astrophys.py

```diff
@@ -145,3 +137,26 @@
 from .utils import generate_unit_summary as _generate_unit_summary
 if __doc__ is not None:
     __doc__ += _generate_unit_summary(globals())
+
+
+# -------------------------------------------------------------------------
+
+def __getattr__(name):
+    if name == "littleh":
+        import warnings
+        from astropy.cosmology.units import littleh
+        from astropy.utils.exceptions import AstropyDeprecationWarning
+
+        warnings.warn(
+            ("`littleh` is deprecated from module `astropy.units.astrophys` "
+             "and may be removed in a future version. "
+             "Use `astropy.cosmology.units.littleh` instead."),
+            AstropyDeprecationWarning)
+
+        return littleh
+
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}.")
+
+
+def __dir__():
```

Comment:
Independently of where the deprecation warning lives, my tendency would be to *not* include deprecated attributes in `__dir__`, i.e., one can get them still, but not be misled into thinking they belong here.

Reply:
Great! I think this was the source of an RTD failure, so removal is fine by me.

Reply:
Done!